chapters/1.md

```diff
@@ -3,37 +3,63 @@
+Both Leo and George have to trust the bank will be honest and will give them their money when they ask and will not go out of business or bankrupt while holding onto their money. Many countries have deposit insurance institutions that may be government run, or a part of the central bank, or even private institutions (one example would be the FDIC in the US). Deposit insurance allows banks to lend out or invest some of the money that they hold and not have to keep the full balance on hand, because if something bad happens where they lose the money, or they go bankrupt, the deposit insurance will make sure that everyone still gets their money back from the bank; this is called *fractional-reserve banking* and it is what is used over most of the world. So, Leo and George can trust their bank as long as they trust the government or institution that insures the money in the bank. In some places, under corrupt, or inefficient governments, this could be a bad idea, but non-the-less they have no choice if they wish to move money across large distances in the traditional banking system they currently have.
 
-Within a distributed network, if you have many disparate parties with no central authority, how do you ensure they all agree? The blockchain is a way of maintaining consensus in a decentralised distributed network. Transactions are grouped into “blocks”, which are linked and secured using cryptography, and every time a new block is solved (i.e. checked, signed, and verified by network participants), these transactions are recorded on the blockchain in chronological order. What goes into a block is just data, but it can be anything. The most obvious example is financial transactions.
+So, what if Leo and George *don't* have the same bank? How do they transfer money? Let's say that Leo banks with Chase Bank, and George Banks with USAA. From Leo's perspective, Leo would need to do the same thing; Leo would ask Chase Bank to reduce the balance in his account and transfer money to George's bank account at USAA where they would increase the balance by the same amount and be updated on the *other* bank's central database. But how does this happen? Why would USAA agree to owe George more? How does this work if the banks don't *trust* each other? Certainly it's possible that after they credit George's account at USAA, the other bank might go out of business and not be able to pay them the $2500 that they allowed George to have. They would be increasing their liability to George, but they would want to decrease their liability to someone else, so that it balances out. The answer would then be for Chase Bank to have an account with USAA, that way when they change the IOU in their central database at USAA, they are reducing the balance that they owe to Chase Bank and they're increasing the amount that they owe George. This arrangement is what is called correspondent banking and it allows the banks to facilitate payments between their respective customers.
```

Comment:
Note for me:
Leo.. Chase Bank
George.. USAA (could we change this since it's too similar to USA ?)

Reply:
Yeah definitely we can change it. I just used the names of the banks most familiar to me.